docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -6,14 +6,389 @@ The parent key for all of the following parameters is `rook_ceph`.
+  nodeSelectorTerms:
+    - matchExpressions: <1>
+        - key: NODE_SELECTOR_KEY
+          operator: Exists
+        ...
+----
+<1> The component creates an entry in `matchExpressions` with `key` equal to the node selector key and `operator=Exists` for each key in parameter <<_node_selector,`node_selector`>>.
+
+* `placement.all.tolerations` is set to the value of parameter <<_tolerations,`tolerations`>>.
+* The component creates as single entry for `storage.storageClassDeviceSets` based on values given in parameter <<_ceph_cluster,`ceph_cluster`>>.
+Users are encouraged to use the parameter <<_ceph_cluster,`ceph_cluster`>> to configure the Ceph cluster's backing storage.
++
+The component expects that the provided storageclass for the backing storage supports `volumeMode=Block`.
+
+See https://rook.io/docs/rook/v1.6/ceph-cluster-crd.html#settings[the Rook.io `CephCluster` documentation] for a full list of configuration parameters.
+
+== Example configurations
+
+=== Configure and tune Ceph's backing storage
+
+To configure the component for an infrastructure which provides backing storage as storageclass `localblock-storage` and uses SSDs to provide local volumes the following config can be provided.
+
+[source,yaml]
+----
+parameters:
+  rook_ceph:
+    ceph_cluster:
+      block_storage_class: localblock-storage
+      tune_fast_device_class: true
+----
+
+=== Configure target size of RBD block pool
+
+To tell the Ceph cluster that the default RBD block pool, which is named `storagepool`, is expected to take up 80% of the Ceph cluster's capacity, the following config can be provided.
+
+[source,yaml]
+----
+parameters:
+  rook_ceph:
+    ceph_cluster:
+      storage_pools:
+        rbd:
+          storagepool:
+            config:
+              parameters:
+                target_size_ratio: "0.8"
+----
+
+=== Configure the component for SElinux-enabled cluster nodes
+
+The component automatically configures the operator on OpenShift 4.
+However, on other Kubernetes distributions on nodes which use SElinux, users need to enable `hostpathRequiresPrivileged` in the operator's helm values.
+
+[source,yaml]
+----
+parameters:
+  rook_ceph:
+    operator_helm_values:
+      hostpathRequiresPrivileged: true <1>
+----
+<1> The operator needs to be informed that deployments which use `hostPath` volume mounts need to run with `privileged` security context.
+This setting is required on any cluster which uses SELinux on the nodes.
+
+=== Disable machine disruption budgets on OpenShift 4
+
+By default, The component enables machine disruption budget management on OpenShift 4.
+Machine disruption budget management can be disabled by providing the following config.
 
 [source,yaml]
 ----
-namespace: example-namespace
+parameters:
+  rook_ceph:
+    cephClusterSpec:
+      disruptionManagement:
+        manageMachineDisruptionBudgets: false <1>
 ----
+<1> Disable machine disruption budget management in the operator.
```

Comment:
Maybe make a how-to page dedicated for OCP4?

Reply:
We can do that in one of the follow-up stories.

Reply:
Same here: just a small page with just this section. Personally, I wouldn't search for examples in the reference, but rather in How-to pages.

Your choice :)

Reply:
done in latest version